src/main.js

```diff
@@ -0,0 +1,131 @@
+import './styles.css'
+// const url= 'http://localhost:3000'
+const url= 'https://omer-phonebook.herokuapp.com'
+//elements
+const root = document.getElementById('root');
+
+// Creating Elements //
+//get All
+const getAllBtn = createElement('button', 'axiosBtn')
+getAllBtn.textContent = 'Get All'
+//phoneBook info
+const getPhoneInfoBtn = createElement('button', 'axiosBtn')
+getPhoneInfoBtn.textContent = 'Get Info'
+//userInfo
+const userIdInput = createElement('input', 'userIdInput')
+userIdInput.placeholder = 'Enter User ID'
+const getUserInfoBtn = createElement('button', 'axiosBtn')
+getUserInfoBtn.textContent = 'Get User Info'
+//createUser
+const createBtn = createElement('button', 'axiosBtn')
+createBtn.textContent = 'Create User'
+//deleteUser
+const deleteBtn = createElement('button', 'axiosBtn')
+deleteBtn.textContent = 'Delete User'
+
+//outputSect
+const outputSect = createElement('div', "ouputSect");
+
+root.append(getAllBtn, getPhoneInfoBtn, getUserInfoBtn , createBtn , deleteBtn, userIdInput, outputSect);
+
+// Event Listeners
+getAllBtn.addEventListener('click',async ()=>{
+    deleteForm();
+    const data = await axios.get(`${url}/api/persons`);
+    outputSect.textContent = 'Contact List';
+    const table = createTable(data.data);
+    outputSect.append(table);
+})
+getPhoneInfoBtn.addEventListener('click',async ()=>{
```

Comment:
your variable names are descriptive but your comments are not, you made a single comments to indicate event listeners,
and you didn't write individual comments to describe the actual functions.
I would suggest writing more comments,I don't think you can go wrong with comments, they could help you a lot in the future.
even if oyu won't come back to this specific project, the habbit of writing a lot of comments is very positive in my opinion